tests/test_actions.py

```diff
@@ -635,7 +635,7 @@ def new_method(self) -> bool:
 
     def test_positional_arg_warns(self) -> None:
         with pytest.warns(DeprecationWarning):
-            HoldDown(Keys.LEFT_ALT, True)
+            HoldDown(None, True)
```

Comment:
I think you should leave these in. It's closer to real life.

Reply:
The problem I saw was you don't typically hold Left alt AND right mouse button.